pymaker/__init__.py

```diff
@@ -681,6 +733,69 @@ def invocation(self) -> Invocation:
         return Invocation(self.address, Calldata(self._contract_function()._encode_transaction_data()))
 
 
+class RecoveredTransact(Transact):
+    def __init__(self, web3: Web3,
```

Comment:
To improve comprehension and for consistency with the other classes like `Receipt`, `Transact` and `Transfer`, I suggest adding a short comment, describing the how, what, why of this class.

Reply:
I added documentation for `get_pending_transactions` method and the `RecoveredTransact` class.